### DISCUSSION TITLE
Disable menu active/hover states

### DISCUSSION THREAD

**Discussion by nandi95** [saadeghi/daisyui#2998] [Ideas / feature requests]
The menu component forces styles hover and active styles on the content of li elements:
<img width="347" alt="Screenshot 2024-04-24 at 18 52 24" src="https://github.com/saadeghi/daisyui/assets/41805560/751532af-a8cb-40c5-a3e9-64d98d08981f">
While this is desired in majority of cases, it would be nice to have the option for disabling this.

**Comment by saadeghi** [comment]
Here's an example: https://play.tailwindcss.com/KuTnffv308
